finetuning/generalists/training/electron_microscopy/mito_nuc/obtain_mito_nuc_em_datasets.py

```diff
@@ -0,0 +1,168 @@
+    )
+    print("All the datasets are available for RoI splitting")
+
+
+def get_concat_mito_nuc_datasets(input_path, patch_shape, with_cem=False):
+    _check_dataset_available_for_rois(path=input_path, patch_shape=patch_shape)
+
+    sampler = MinInstanceSampler()
+    standard_label_trafo = PerObjectDistanceTransform(
+        distances=True, boundary_distances=True, directed_distances=False,
+        foreground=True, instances=True, min_size=25
+    )
+
+    # mitoem parameters
+    mitoem_train_rois = [np.s_[100:110, :, :], np.s_[100:110, :, :]]
+    mitoem_val_rois = [np.s_[0:5, :, :], np.s_[0:5, :, :]]
+
+    # platynereis cell dataset parameters
+    platy_root = os.path.join(input_path, "platynereis")
+
+    # platynereis nuclei dataset parameters
+    platy_nuclei_template = "nuclei/train_data_nuclei_%02i.h5"
+    platy_nuclei_label_key = "volumes/labels/nucleus_instance_labels"
+
+    platy_nuclei_train_samples = [1, 2, 3, 4, 5, 6, 7, 8]
+    platy_nuclei_train_rois = compute_platy_rois(platy_root, platy_nuclei_train_samples, ignore_label=-1,
+                                                 file_template=platy_nuclei_template, label_key=platy_nuclei_label_key)
+    platy_nuclei_val_samples = [9, 10]
+    platy_nuclei_val_rois = compute_platy_rois(platy_root, platy_nuclei_val_samples, ignore_label=-1,
+                                               file_template=platy_nuclei_template, label_key=platy_nuclei_label_key)
+
+    def mitoem_dataset(split, roi_choice):
+        return datasets.get_mitoem_dataset(
+            path=os.path.join(input_path, "mitoem"), splits=split, download=True, patch_shape=patch_shape,
+            rois=roi_choice, label_transform=standard_label_trafo, ndim=2, raw_transform=identity,
+            sampler=MinInstanceSampler(min_num_instances=5)
+        )
+
+    mitoem_train_dataset = mitoem_dataset("train", mitoem_train_rois)
+    mitoem_val_dataset = mitoem_dataset("val", mitoem_val_rois)
+
+    def platy_nuclei_dataset(roi_choice, sample_ids):
+        return datasets.get_platynereis_nuclei_dataset(
+            path=platy_root, patch_shape=patch_shape, download=True, sampler=sampler, ndim=2,
+            label_transform=ResizeLabelTrafo(patch_shape[1:]), rois=roi_choice,
+            raw_transform=ResizeRawTrafo(patch_shape[1:]), sample_ids=sample_ids
+        )
+
+    platy_nuclei_train_dataset = platy_nuclei_dataset(platy_nuclei_train_rois, sample_ids=platy_nuclei_train_samples)
+    platy_nuclei_val_dataset = platy_nuclei_dataset(platy_nuclei_val_rois, sample_ids=platy_nuclei_val_samples)
+
+    train_datasets = [mitoem_train_dataset, platy_nuclei_train_dataset]
+    val_datasets = [mitoem_val_dataset, platy_nuclei_val_dataset]
+
+    if with_cem:
+        def cem_dataset(split):
+            return datasets.cem.get_mitolab_dataset(
+                path=os.path.join(input_path, "mitolab"), split="train", val_fraction=0.1
```

Comment:
You have `split="train"` hard-coded here, but it should be `split=split`!!

Reply:
Fixed it, thanks!